client/my-sites/themes/theme-showcase.jsx

```diff
@@ -159,7 +159,20 @@ class ThemeShowcase extends Component {
 
 	scrollToSearchInput = () => {
 		if ( ! this.props.loggedOutComponent && this.scrollRef && this.scrollRef.current ) {
-			this.scrollRef.current.scrollIntoView();
+			// If you are a larger screen where the theme info is displayed horizontally.
+			if ( window.innerWidth > 600 ) {
+				return;
+			}
+			const headerHeight = document.getElementById( 'header' )?.getBoundingClientRect().height;
+			const screenOptionTab = document
+				.getElementById( 'screen-options-tab-id' )
+				?.getBoundingClientRect().height;
+
+			const yOffset = -( headerHeight + screenOptionTab ); // A number that takes the Adminbar as well as the screen options into account on mobile view ports.
```

Comment:
Thanks, that makes a lot sense now 👍 

I'd suggest shortening the comment a bit if possible to keep the line shorter, maybe something like:

```suggestion
			const yOffset = -( headerHeight + screenOptionTab ); // Total height of admin bar and screen options on mobile.
```